rasterio/rio/features.py

```diff
@@ -129,7 +142,8 @@ def __call__(self):
                     yield {
                         'type': 'Feature',
                         'id': str(i),
-                        'properties': {'val': i},
+                        'properties': {
+                            'val': i, 'filename': os.path.basename(src.name)},
```

Comment:
@sgillies totally minor nit, would you mind putting the closing brace on a newline for readability?